x-pack/plugins/enterprise_search/public/applications/app_search/components/relevance_tuning/boosts/boost_item_content/boost_item_content.tsx

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+
+import { useActions } from 'kea';
+
+import { EuiButton, EuiFormRow, EuiPanel, EuiRange, EuiSpacer } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+
+import { RelevanceTuningLogic } from '../..';
+import { Boost, BoostType } from '../../types';
+
+import { FunctionalBoostForm } from './functional_boost_form';
+import { ProximityBoostForm } from './proximity_boost_form';
+import { ValueBoostForm } from './value_boost_form';
+
+interface Props {
+  boost: Boost;
+  index: number;
+  name: string;
+}
+
+export const BoostItemContent: React.FC<Props> = ({ boost, index, name }) => {
+  const { deleteBoost, updateBoostFactor } = useActions(RelevanceTuningLogic);
+  const { type } = boost;
+
+  const getBoostForm = () => {
+    switch (type) {
+      case BoostType.Value:
+        return <ValueBoostForm boost={boost} index={index} name={name} />;
+      case BoostType.Functional:
+        return <FunctionalBoostForm boost={boost} index={index} name={name} />;
+      case BoostType.Proximity:
+        return <ProximityBoostForm boost={boost} index={index} name={name} />;
```

Comment:
Just wanted to shout this out - thanks so much for breaking this up into various files/components. Makes it super easy to read/review/test just IMO! Super appreciated!